cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_detail.spec.js

```diff
@@ -144,5 +146,115 @@ if (Cypress.env('WORKSPACE_ENABLED')) {
         });
       });
     });
+
+    if (Cypress.env('SECURITY_ENABLED')) {
+      describe('update with different workspace access level', () => {
+        let originalUser = ADMIN_AUTH.username;
+        let originalPassword = ADMIN_AUTH.password;
+        beforeEach(() => {
+          originalUser = ADMIN_AUTH.username;
+          originalPassword = ADMIN_AUTH.password;
+        });
```

Comment:
Do we actually need to update `originalUser` and `originalPassword` in before each? Shall we simply do a `const`
```suggestion
        const originalUser = ADMIN_AUTH.username;
        const originalPassword = ADMIN_AUTH.password;
```